src/Microsoft.AspNet.Authentication.OAuth/OAuthAuthenticationMiddleware.cs

```diff
@@ -43,27 +43,27 @@ public class OAuthAuthenticationMiddleware<TOptions, TNotifications> : Authentic
             // todo: review error handling
             if (string.IsNullOrWhiteSpace(Options.AuthenticationScheme))
             {
-                throw new ArgumentException(string.Format(CultureInfo.CurrentCulture, Resources.Exception_OptionMustBeProvided, nameof(Options.AuthenticationScheme)));
+                throw new ArgumentException(string.Format(CultureInfo.CurrentCulture, Resources.Exception_OptionMustBeProvided, nameof(Options)));
```

Comment:
Will this change result in the user being informed that 'Options.AuthenticationScheme' is missing or "Options".
What does the exception message look like?